cluster/gce/trusty/node.yaml

```diff
@@ -133,12 +131,15 @@ Content-Disposition: attachment; filename="kube-install-additional-packages.conf
 
 description "Install additional packages used by kubernetes"
 
-start on started docker
+start on stopped kube-install-packages
 
 script
```

Comment:
Is this a bash script? sh script? For our bash scripts, we tend to set

```
set -o errexit
set -o nounset
set -o pipefail
```

at the top as it helps catch a bunch of programming errors (admittedly, it only catches them at runtime, but still).